index.bs

```diff
@@ -905,6 +905,49 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+
+### Inform user platform authenticator capability - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorReady">
+This method can be used by [=[RPS]=] to inform user they can create a new credential with one or more platform authenticators.
+The method gives [=[RPS]=] the ability to inform the client that they support registering credentials with <a>platform 
+authenticators</a> while giving users control over whether they want to share this information. When the 
```

Comment:
Recommend: "while giving users control over whether this information is appropriate to provide to the RP."